corehq/apps/users/models.py

```diff
@@ -3120,6 +3120,17 @@ def _generate_key(self):
         new_uuid = uuid4()
         return hmac.new(new_uuid.bytes, digestmod=sha1).hexdigest()
 
+    # Remove this after key fields are deleted and verified no errors occur
+    @property
+    def key(self):
+        logging.warning("Attempted to access api key directly for %s", self.name)
+        return self.plaintext_key
+
+    @key.setter
+    def key(self, value):
+        logging.warning("Attempted to set api key directly for %s", self.name)
```

Comment:
Does a `warning` level log show up in sentry?  Might also try a `soft_assert` for these, so there's exponential backoff

Reply:
you're right, it doesn't look like it shows up in sentry. Do you know where it does show up? Anyhow, I changed it to soft_asswert  [440bbfc](https://github.com/dimagi/commcare-hq/pull/35680/commits/440bbfce73560b9e2f345fdc150998a1778412da)